node-module-system/calculate.js

```diff
@@ -0,0 +1,32 @@
+import add from './add.js';
+import subtract from './subtract.js';
+import multiply from './multiply.js';
+import divide from './divide.js';
+
+const inputs = [process.argv[3], Number(process.argv[2]), Number(process.argv[4])];
+
+const runTime = (array) => {
+  let value = null;
+  switch (array[0]) {
+    case 'plus':
+      value = add(array[1], array[2]);
+      break;
+    case 'minus':
+      value = subtract(array[1], array[2]);
+      break;
+    case 'times':
+      value = multiply(array[1], array[2]);
+      break;
+    case 'over':
+      value = divide(array[1], array[2]);
+      break;
+    default:
+      console.log('Please input a number followed by an operation followed by another operation. Acceptable operators are: plus, minus, times and over.');
+  }
+  if (value) {
+    console.log(value);
+  }
+
+};
+
+runTime(inputs);
```

Comment:
I like doing the calculations in a function (though I would name it `calculate`).

A better separation of concerns, though, would be to have your function do the calculating and the logging out here. That would mean your code looks like:
* Handle user input
* Do the logic
* Handle user output
In addition to being easier to read, it makes it easier to change your mind about the user interface (what if you decided to build a GUI on this instead of a CLI?).